app/services/course/assessment/programming_evaluation_service.rb

```diff
@@ -107,7 +108,7 @@ def execute
   def create_evaluation
     Course::Assessment::ProgrammingEvaluation.create(
       course: @course, language: @language, package_path: @package, memory_limit: @memory_limit,
-      time_limit: @time_limit.to_i
+      time_limit: @time_limit
```

Comment:
Ok!

But wait, this is CPU time ? 90S CPU time is quite long, I thought it's a overall time so suggested 90.